boards/kernel_layout.ld

```diff
@@ -226,7 +216,7 @@ SECTIONS
 
         . = ALIGN(4);
         _erelocate = .;
-    } > ram AT > rom
+    } > ram
```

Comment:
This is a regression too, and a pretty serious one. This syntax ensures that this section is charged to both the `ram` and `rom` sections. The `AT` syntax simply places it there without accounting for size. This silent failure was the root cause of a pretty awful to track down signpost bug.

Reply:
`AT > rom` doesn't seem to work with lld. Instead, I added an assert to the bottom of the linker script. Take a look and see if that would also catch the error.

Reply:
Haven't tested, but shouldn't that be `< LENGTH(rom)` not `> LENGTH(rom)`?

Reply:
It only compiles this way.